### DISCUSSION TITLE
screenOptions header?

### DISCUSSION THREAD

**Discussion by compojoom** [react-navigation/react-navigation#9621] [Q&A]
I'm trying to add a sticky view right after the header on all screens in our app. 
The component looks like this:
```
import React from "react"
import { Header as OriginalHeader } from "@react-navigation/elements"
import OfflineError from "components/common/OfflineError"

/**
 * Wrap the original RN-header and display network related errors
 * below it
 *
 * @param props
 * @returns {*}
 * @constructor
 */
export const Header = props => {
  return (
    <>
      <OriginalHeader {...props} />
      <OfflineError delay={5000} />
    </>
  )
}

```

in version 4 I was able to just do:
```
defaultNavigationOptions: ({ navigation }) => ({

header: props => <Header {...props} />,
})
```
and it was all good.

But now with v6 if I use screenOptions with `header: props => <Header {...props} />,`

I get an empty header? Can't I just pass the whole props object to Header or do I really have to calculate all the headerLeft, right, etc myself?

**Comment by raajnadar** [comment]
If you see the `console.log` of the props you can props changed between v5 & v6. Make sure you read the migration guide and make changes accordingly.
